### DISCUSSION TITLE
笔记能否增加2个功能：

### DISCUSSION THREAD

**Discussion by zhangyan000005** [windingwind/zotero-better-notes#1043] [Ideas]
一是对于标题下面内容增加收起功能，点击标题后可收起，二是可通过点击大纲直接定位笔记信息
因为当笔记过长时，需要下拉很久才能定位到需要修改的位置，有点麻烦

**Comment by windingwind** [comment]
1. 笔记内的段落折叠无法实现
2. 点击大纲可跳转笔记对应行
